inscriptions/ISic001383.xml

```diff
@@ -193,29 +198,23 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140889">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — — — — — — — — —</supplied>
-                    <lb n="2"/>
-                    <supplied reason="lost">— —</supplied>ϹΙΑ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="3"/>ἔτη Γ
-                    <supplied reason="lost">․ʹ τελ(ευτᾷ) δὲ τῇ</supplied>
-                    <lb n="4"/>πρὸ ϛ καλ<supplied reason="lost">ανδῶν — —</supplied>-<lb n="5"/>βρίῳ ΜΝ
-                    <supplied reason="lost">— — — — — —</supplied>
-                    <lb n="6"/>
+                    <lb n="1"/><gap reason="lost" quantity="1" unit="line"/>
+                    <lb n="2"/><gap reason="lost" extent="unknown" unit="character"/><orig>ΣΙΑ</orig><gap reason="lost" extent="unknown" unit="character"/>
+                    <lb n="3"/>ἔτη <num value="3">Γ</num><supplied reason="lost">. <expan><abbr>τελ</abbr><ex>ευτᾷ</ex></expan> δὲ τῇ</supplied>
```

Comment:
IG wouild put this on the start of line 4 I think, since clearly more is missing left than right? Lose the full-stop also.